include/ttmlir/Dialect/CMakeLists.txt

```diff
@@ -3,3 +3,4 @@ add_subdirectory(TTIR)
 add_subdirectory(TTNN)
 add_subdirectory(TTMetal)
 add_subdirectory(TTKernel)
+add_subdirectory(LLVM)
```

Comment:
Note: imo this directory structure is most sensible.  It's a little ugly to add a dir for "Dialect/LLVM" because LLVM isn't a new Dialect, and we only want to define a single Transform which is (softly) dependent on other TTIR passes to create module + funcs with expected attrs.  On the other hand, jamming this Transform into TTIR or some other existing dir feels even worse to me, because this is strictly an LLVMDialect -> LLVMDialect Transform

Reply:
Yeah I'm not sure what's conventional here, @sdjordjevicTT, was anything like this covered at MLIR conf?

I generally agree with you though could see a loose argument for putting it in TTIR since it's defining the TTIR->CPU calling convention.

Reply:
I am honestly not sure, we didn't come up with something similar during the conference. I was observing a few mlir repos, and none of them had LLVM-specific folders in them, however not sure if any of them had something similar to implement.

Maybe the best advice we can get on this topic is their discord channel. @vwellsTT you can ask them this specific question here:
https://discord.com/channels/636084430946959380/642426447167881246

They are mostly responsive and helpful :)

Reply:
Ok--the discord link doesn't seem to take me to a channel properly though for some reason.  Can you give me name or something?

Reply:
<img width="1296" alt="image" src="https://github.com/user-attachments/assets/d8c850f6-b4f4-4ee8-baa9-df1d0f2dd7f3" />

Reply:
Ah, nvm, needed to join LLVM group first, now it works I think